src/UI/Implementation/Component/Dropdown/Renderer.php

```diff
@@ -25,6 +31,8 @@ protected function renderDropdown(Component\Dropdown\Dropdown $component, Render
 		$tpl_name = "tpl.standard.html";
 		$tpl = $this->getTemplate($tpl_name, true, true);
 
+		$tpl->setVariable("CLASSNAME", $this->class_name);
```

Comment:
Nasty. Somebody watching at the template has no clue what "CLASSNAME" will be set to by PHP.